src/screens/StartScreen.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { StyleSheet, View, Button, Image } from 'react-native'
+
+const StartScreen = ({ navigation }) => {
+  return (
+    <View style={styles.container}>
+      <Image style={styles.eightBall} source={require('../../assets/ate-ball-home.png')}/>
+      <View style={styles.startBtn}>
+        <Button
+          title="Get Shakin'"
+          onPress={() => navigation.navigate('Form')}
+        />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    // paddingTop: 50,
```

Comment:
👀👀

Reply:
I forgot to delete this comment but the padding was commented out due to stack navigation having its own styling for the header which resulted in the padding being unnecessary